Sources/Sentry/SentryFileIOTrackingIntegration.m

```diff
@@ -32,8 +32,8 @@ - (BOOL)installWithOptions:(SentryOptions *)options
 
 - (SentryIntegrationOption)integrationOptions
 {
-    return kIntegrationOptionEnableSwizzling | kIntegrationOptionIsTracingEnabled
-        | kIntegrationOptionEnableAutoPerformanceTracing | kIntegrationOptionEnableFileIOTracing;
+    return kIntegrationOptionIsTracingEnabled | kIntegrationOptionEnableAutoPerformanceTracing
+        | kIntegrationOptionEnableFileIOTracing;
```

Comment:
Why are we removing `kIntegrationOptionEnableSwizzling` here?

Reply:
Because if it stays there, people can not use the manual methods introduced in #4862 and #4863

Reply:
I added a comment to the PR with details.